src/java/fr/paris/lutece/plugins/appointment/modules/ants/utils/PredemandeCodeUtils.java

```diff
@@ -37,16 +37,78 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import org.apache.commons.lang3.StringUtils;
+
+import fr.paris.lutece.portal.service.util.AppPropertiesService;
+import fr.paris.lutece.portal.util.mvc.utils.MVCUtils;
+import fr.paris.lutece.util.url.UrlItem;
+
 public class PredemandeCodeUtils {
+	
+	private static final String PROPERTY_SESSION_ATTRIBUTE_NAME =AppPropertiesService.getProperty("ants.session.attribute.name");
```

Comment:
this is not a property, you should declare the property first, and get the value as constant

private static final String PROPERTY_PREDEMANDE_CODE_LIST_SESSION_ATTRIBUTE_NAME_KEY ="ants.session.attribute.name";
private static final String CONSTANT_PREDEMANDE_CODE_LIST_SESSION_ATTRIBUTE_NAME =  =AppPropertiesService.getProperty(PROPERTY_PREDEMANDE_CODE_LIST_SESSION_ATTRIBUTE_NAME_KEY);